### PR TITLE
fix(rich-text-editor): NO-JIRA fix error when allowBulletList false

### PR DESCRIPTION
**Pull request by braddialpad**
# fix(rich-text-editor): fix error when allowBulletList false

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [x] Fix

## :book: Description

Don't trigger `splitListItem()` on enter when `allowBulletList` is false

## :bulb: Context

If allowBulletList was false it would not include the list item extension and therefore error out on the splitListItem command. This will prevent calling this function if allowBulletList is false.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

For all Vue changes:

- [x] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script. Read docs here: [Dialtone Vue Sync Script](../packages/dialtone-vue3/.github/CONTRIBUTING.md#dialtone-vue-sync-script)

## :crystal_ball: Next Steps

Update product and CP to beta

### CODE REVIEW

File: packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue [hunk 1]
```diff
@@ -397,7 +397,7 @@ export default {
               }
               editor.commands.first(({ commands }) => [
                 () => commands.newlineInCode(),
-                () => commands.splitListItem('listItem'),
+                () => self.allowBulletList ? commands.splitListItem('listItem') : false,
```
Comment (juliodialpad) [inline packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue]: Does this suggestion work? I like it more.

Or is it necessary to return that false?

```suggestion
                () => self.allowBulletList && commands.splitListItem('listItem'),
```

Comment (braddialpad) [reply packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue]: Should work fine it's the exact same logic. Still returns false if allowBulletList is false.

**Review by juliodialpad — APPROVED** [review]
Just update on Vue 3 before merging, thanks!

**Comment by braddialpad** [comment]
> Just update on Vue 3 before merging, thanks!

thanks almost forgot this.

**Pull request merged**